eahub/profiles/tests/test_profiles_models.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+from model_mommy import mommy
+
+from eahub.profiles.models import Profile
+
+
+class ProfileTests(TestCase):
+    def setUp(self):
+        self.name = "Test Person"
+
+    def test_profile_creation(self):
```

Comment:
This is effectively testing the behavior of Model Mommy; I don't think it adds any coverage to our code.